src/Routes/Login.js

```diff
@@ -1,51 +1,30 @@
 import React from "react";
-import AppButton from "../UI/Button";
-import GenericInput from "../UI/Input";
-import Modal from "../UI/Modal";
+import styled from "styled-components";
 import NavBar from "../UI/NavBar";
-import { UserOutlined, KeyOutlined } from "@ant-design/icons";
-
-const iconStyling = {
-  color: "#c1c1c1",
-  fontSize: "18px",
-  marginLeft: "8px",
-};
-
+import LoginModal from "../UI/LoginModal";
+import BlurDiv from "../UI/BlurDiv";
+import backgroundImage from "../StarsBG.jpg";
+const StyledImage = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+`;
 function Login() {
   return (
-    <div className="App">
+    <div style={{ height: "100%" }}>
       <NavBar
-        username={"Fady Philips"}
-        showMyListIcon={true}
-        showSearchbar={true}
+        username={null}
+        showMyListIcon={false}
+        showSearchbar={false}
         showLoginButton={false}
         showSignUpButton={true}
         showLogOutButton={false}
       />
-      <Modal closeOnClick={() => console.log("object")}>
```

Comment:
is it important to have this line here 
**closeOnClick={() => console.log("object")}**

Reply:
it isn't there. that is one of the lines that were removed